### PR TITLE
CW-3 identity model access

### PR DESCRIPTION
**Pull request by yuramorozov01**
Was made access and CRUD operations in database with:
1. User information.
2. Role information.
3. Token information.

### CODE REVIEW

**Review by ArtyomButrim — APPROVED** [review]
LGTM.

File: src/application/common/interfaces/genericRepository.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default interface RepositoryInterface<T> {
```
Comment (MrKonstantinSh) [inline src/application/common/interfaces/genericRepository.ts:1]: Naming conventions

**Review by MrKonstantinSh — APPROVED** [review]
While LGTM

**Review by axcxdxbx — APPROVED** [review]
^_^

**Pull request merged**